common/models/middleware.js

```diff
@@ -93,13 +92,13 @@ function ready(Middleware) {
         }
       }
       var phasePlaceHolder =
-      {
-        phase: phase,
-        isPhasePlaceHolder: true,
-        order: order,
-        facetName: facetName,
-        name: '*' + phase // Set the name to be unique
-      };
+        {
```

Comment:
I personally prefer this style:

```js
var phasePlaceHolder = {
  phase: phase,
  // ...
};
```

It should also minimise the amount of changes needed to make eslint happy.